plugins/fabrik_list/inlineedit/inlineedit.js

```diff
@@ -170,40 +198,32 @@ var FbListInlineEdit = new Class({
 			}
 			if (typeOf(this.td.getPrevious()) === 'element') {
 				e.stop();
-				this.select(e, this.td.getPrevious());
+				this.select(e, this.getPreviousEditable(this.td));
 			}
 			break;
 		case 40:
 			//down
-			if (this.inedit) {
-				return;
+			if (this.inedit && this.options.tabSave) {
```

Comment:
how does the tab save option work if you are editing a textarea? Im not sure closing the inline editor on tab would be appropriate in this case?